src/combinatorics/simple_graph/basic.lean

```diff
@@ -576,4 +504,49 @@ end
 
 end complement
 
+section maps
+
+/-- A graph homomorphism is a map on vertex sets that respects adjacency relations -/
+abbreviation homomorphism (G G' : simple_graph V) := rel_hom (G.adj) (G'.adj)
+
+infix ` →g ` : 50 := homomorphism
```

Comment:
I think `hom` and `iso` match the naming convention better, rather than `homomorphism` and `isomorphism`.

Reply:
ok will use `hom` and `iso`